pallets/handles/src/handles-utils/build.rs

```diff
@@ -32,17 +39,25 @@ fn main() {
 		// The first character in the line will be the value of each key
 		let normalized_character = original_line_characters.next().unwrap();
 
+		// Skip if the normalized is not in the allowed range
+		if !allowed_char(normalized_character) {
```

Comment:
So I'm guessing that the first character (representation) is enough to skip the whole set

Reply:
Right. If the normalized character is not allowed, it will just be rejected as we test that the canonical only has allowed characters.